README.md

```diff
@@ -28,7 +28,7 @@ Conductor Server is a [Spring Boot](https://spring.io/projects/spring-boot) proj
 #### Development
 The server can be started locally by running `./gradlew bootRun` from the project root. This will start up Conductor with an in-memory persistence and queue implementation. It is not recommended for production use but can come in handy for quickly evaluating what Conductor's all about. For actual production use-cases, please use one of the supported persistence and queue implementations.
 
-You can verify the development server is up by navigating to `http://localhost:8080/swagger-ui/index.html` in a browser.
```

Comment:
http://localhost:8080/swagger-ui/index.html opens Swagger UI with the sample PetStore API specification which might be a bit confusing.